blog/_posts/2018-09-17-Code-2-a-good-summer.md

```diff
@@ -0,0 +1,138 @@
+
+If you get the chance to apply for RGSoC do it. You won't regret it.
+
+To learn it takes patience, support and space to grow. We have had all 3 of these during RGSoC.
+
+We all learn at a different pace, in different ways but we all have something to offer and we learn more when we share with each other. RGSoC has a great community and alumni.
+
+## Life after RGSoC
+
+Georgina has gained in confidence to learn more in the field of software engineering, give more talks and continue to inspire other women and girls into software engineering. The next steps for Georgina, is to attend the React.js conference in Verona in October, look at attending the Node.js conference. Georgina is on the technical path of the toastmaster course to gain more skills in talking in public and to larger audiences. Georgina will also create and pitch her MVP app and present for investment and funding in 2019.
+
+Eva will continue to divide her time between coding and GirlsCode MK including organising Django Girls MK, CryptoParty MK and Ada Lovelace Wikipedia-edit-a-thon. GirlsCode MK has also been accepted to a local accelerator starting October - so that we can make it even better and bigger in the future! With her new-gained coding skills, Eva has already landed a couple of projects that she will take on after RGSoC. And, of course, there’s Bahmni!
+
+We are both going to do a demo of our project at Thought Works office in London and will be streamed to their Berlin office on 2 November.
+
+## Friends Made
+
+Other teams of RGSoC on slack, coaches, mentors and organisers of RGSoC, we have your links on social media and hope to continue to follow you.
+
+### Last post would not be complete without ...
+
+**THANK YOUS**
+
+Thank you to
+Supervisor:
+
+Ramon who met us weekly, checking in and making sure our welfare was good. He was always positive, we loved his support and smiles on a Monday morning - far better than any coffee!
+
+Mentors:
+
+Ivo - for his unfailing guidance, the right balance of learning but also guiding so we didn’t become overwhelmed. His weekly meetings, feedback on our code and management of the project. His dedication to writing stories and providing us with information even during his wedding season! Congratulations to his friends that got married over summer.
+
+Wolf - for such a warm introduction and getting us started with the Bahmni project especially giving up your lunch times.
+
+Coaches:
+
+Neil - for all his knowledge and his fabulous git workshop
+Kalan - for his guidance especially on testing
+PJ and Rob for helping during girls code sessions.
+
+RGSoC Organisers : for setting up RGSoC meetups, organising tickets for conferences, all the admin work that goes into organising these summers. You Rock!!
+
+Anna
```

Comment:
Hi, 'Anna' is there for 2 times. Did you mean 'Ana Sofia'? And did you also mean 'Vaishali', as I don't any RGSoC organizer named Valisha. :)